src/lib/core/CHIPTLVReader.cpp

```diff
@@ -784,7 +784,7 @@ CHIP_ERROR TLVReader::VerifyElement()
     return CHIP_NO_ERROR;
 }
 
-Tag TLVReader::ReadTag(TLVTagControl tagControl, const uint8_t *& p)
+Tag TLVReader::ReadTag(TLVTagControl tagControl, const uint8_t *& p) const
```

Comment:
Shouldn't this just be a static method?  It does not use any member state at all.

Reply:
It seems to use `ImplicitProfileId`

Reply:
Oh, that's a member variable?  OK, then.